nekoyume/Assets/_Scripts/Blockchain/BlockchainCache.cs

```diff
@@ -0,0 +1,262 @@
+#nullable enable
```

Comment:
Is this class used anywhere yet? Do you have a follow-up PR ready?

Reply:
Yes, there is a follow-up PR. Um.. I adjust some branches and PRs and mention you.

Reply:
It's just a suggestion and you don't have to follow it, but... I think it will be helpful in reviewing this PR if you post a follow-up PR in advance. Even Draft is fine.

Reply:
@riemannulus I opened the follow-up pull request as a draft. https://github.com/planetarium/NineChronicles/pull/3202